package.json

```diff
@@ -54,7 +54,7 @@
     "parseurl": "1.3.2",
     "path-to-regexp": "1.5.3",
     "pluralize": "1.2.1",
-    "prompt": "0.2.14",
+    "prompt": "^1.1.0",
```

Comment:
Since `prompt` is an externally maintained dep, lets use a strict semver range `1.1.0`

Reply:
Okay that works well